### PR TITLE
feature/paginate-survey-responses

### PR DESCRIPTION
**Pull request by Ahmedhossamdev**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ gem "importmap-rails"
 gem "jbuilder", '~>2.12'
 gem 'jsonb_accessor', '~> 1.4'
 gem 'mailgun-ruby', '~>1.2.14'
+gem 'pagy'
```
Comment (aaronbrethorst) [inline Gemfile:13]: please pin this to the major version, i.e.:

`gem 'pagy', '~>9'`

File: app/views/admins/survey_responses/index.html.erb [hunk 2]
```diff
@@ -34,4 +36,8 @@
       </tr>
     <% end %>
   <% end %>
-<% end %>
\ No newline at end of file
+<% end %>
+
```
Comment (aaronbrethorst) [inline app/views/admins/survey_responses/index.html.erb:40]: please delete the blank lines at the end of the file

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -14,6 +14,9 @@
     <meta name="apple-itunes-app" content="app-id=329380089">
     <%= stylesheet_link_tag "tailwind", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag 'pagy.tailwind', media: 'all' %>
```
Comment (aaronbrethorst) [inline app/views/layouts/application.html.erb:17]: are these necessary since you included `pagy.tailwind` in the application.tailwind.css file?

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
looking good! couple minor changes I'd like to see before merging.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (Ahmedhossamdev) [reply app/views/layouts/application.html.erb:17]: When I removed it, the pagy navigation bar was displayed without any style. But I just tried removing the import from application.tailwind.css and kept the pagy.tailwind link_tag application.html, and it works now.

File: Gemfile (hunk 1 above)
Comment (Ahmedhossamdev) [reply Gemfile:13]: Done

File: app/views/admins/survey_responses/index.html.erb (hunk 2 above)
Comment (Ahmedhossamdev) [reply app/views/admins/survey_responses/index.html.erb:40]: Done

**Pull request merged**